app/controllers/admin_v2/profile/password_controller.rb

```diff
@@ -0,0 +1,19 @@
+module AdminV2
+  class Profile::PasswordController < AdminV2::BaseController
+    def update
+      if current_admin_user.update_with_password(password_params)
+        # Sign in the user by passing validation in case their password changed
+        bypass_sign_in(current_admin_user)
```

Comment:
Lifted this from the Devise docs: https://github.com/heartcombo/devise/wiki/How-To:-Allow-users-to-edit-their-password#solution-3